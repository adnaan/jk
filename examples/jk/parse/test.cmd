jk transform --stdout ./add-version.js ./example.yaml
