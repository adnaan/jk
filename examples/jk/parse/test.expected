apiVersion: v1
data:
  config.json: '{"path":"some/path/somewhere","version":2}'
kind: ConfigMap
metadata:
  name: config
